Tools/ssl/multissltests.py

```diff
@@ -70,7 +74,7 @@
 parser = argparse.ArgumentParser(
     prog='multissl',
     description=(
-        "Run CPython tests with multiple OpenSSL and LibreSSL "
+        "Run CPython tests with multiple cryptography libraries"
         "versions."
     )
```

Comment:
Let's slip in some of the nice Python 3.14 improvements :)
```suggestion
    ),
    color=True,
    suggest_on_error=True,
```
https://docs.python.org/3.14/library/argparse.html#suggest-on-error